src/components/Dropdown/Dropdown.tsx

```diff
@@ -96,7 +96,7 @@ export const Dropdown: FC<DropdownProps> = ({
                     data-test-id="dropdown-trigger"
                     className={merge([
                         "tw-overflow-hidden tw-flex-auto tw-h-full tw-rounded tw-text-left tw-outline-none",
-                        size === DropdownSize.Small ? "tw-py-1 tw-px-3" : "tw-p-5 tw-min-h-[60px]",
+                        size === DropdownSize.Small ? "tw-py-2 tw-px-3 tw-min-h-[34px]" : "tw-p-5 tw-min-h-[60px]",
```

Comment:
We don't have the cursor in the trigger of the color picker flyout https://deploy-preview-401--arcade-components.netlify.app/?path=/story/components-color-picker-flyout--flyout

Reply:
Could you elaborate? I tried it out and couldn't replicate the issue.

Reply:
https://deploy-preview-401--arcade-components.netlify.app/?path=/story/components-color-picker-flyout--flyout

On the top near the border, the cursor is not a pointer but click seems to work (Firefox)

Reply:
Ah I see now. You're correct and it's been fixed. 
But I'm not sure if this is the best way to do it since the padding is coming from the wrapping `button` element.
This means that whenever the `MenuItemContent` component is used, we will expect the parent to have the appropriate padding styling.